Sources/WalletConnectEcho/Register/EchoRegisterService.swift

```diff
@@ -1,12 +1,14 @@
 import Foundation
 import WalletConnectNetworking
+import WalletConnectJWT
 
 actor EchoRegisterService {
     private let httpClient: HTTPClient
     private let projectId: String
     private let clientId: String
     private let logger: ConsoleLogging
     private let environment: APNSEnvironment
+    private let echoAuthenticator: EchoAuthenticating
     // DID method specific identifier
     private var clientIdMutlibase: String {
         return clientId.replacingOccurrences(of: "did:key:", with: "")
```

Comment:
may be use DIDKey object?

Reply:
in what way?
actually clientIdStorage could return DIDKey?

Reply:
i mean: 
`let clientId: DIDKey`

and after you can get the string with `did(prefix: Bool, variant: DIDKeyVariant)`

I created DIDKey type to avoid of using string keys. Actually good idea to return DIDKey from clientIdStorage

Reply:
clientID cannot be DIDkey because we could confuse the variant